### ISSUE TITLE
aViewer included with WAT: the labels in the trieeview for the output pane are missing

### ISSUE THREAD

**Issue by matatk** [ThePacielloGroup/WebAccessibilityToolbar#57]
There is a tree view in the top-rigt pane, which displays all the accessibility property names and their values.  In the version of aViewer included with WAT, the names of the properties appear to be missing. (The ARIA property names are there, but not any of the others.)

@stevefaulkner: do you get this problem too? @menovak and I do.

**Comment by stevefaulkner** [comment]
version in repo is borked get latest version will point you to it

**Comment by matatk** [comment]
(Need to re-open this until it's fixed in this repo.)
